### ISSUE TITLE
batch commands send error

### ISSUE THREAD

**Issue by siddontang** [pingcap/go-ycsb#108]
```
./go-ycsb load tikv -p tikv.type=raw
```

When the test finished, it outputs too many `batch commands send error` logs, I remember we have fixed in tidb, seem forget to fix it client-go.

**Comment by siddontang** [comment]
PTAL @disksing

**Comment by hicqu** [comment]
I remember I have fixed it in go-ycsb, when go-ycsb sitll used tidb instead of client-go. Does go-ycsb uses client-go now? If so I think we need to port the latest changes from tidb.

**Comment by disksing** [comment]
Yes, it is using `client-go` now. There is a todo issue in `client-go`: https://github.com/tikv/client-go/issues/19
